src/backend/marsha/core/api.py

```diff
@@ -574,6 +582,8 @@ def update_live_state(
             video.live_info = live_info
             create_mediapackage_harvest_job(video)
             delete_aws_element_stack(video)
+            if settings.LIVE_CHAT_ENABLED:
+                close_room(video.id)
```

Comment:
Why not [destroying](https://xmpp.org/extensions/xep-0045.html#destroyroom) the room instead of restricting its access ? Is it to leave the room active until all users leave the chat ?

It the room is automatically destroyed on the XMPP server when the last member leaves it ?

Reply:
We want to keep the room open to use it by the instructor after. To opy some unanswered questions for example.

Reply:
Ok, I see :+1: 
Do you confirm that the room is destroyed on the XMPP server when everyone has left ?
I just want to make sure there's no kind of memory leak on the server because of this.

Reply:
The room is persistent so no the room will stay. But the message conservation has a deadline, it's called muc archive. To the message will be erased after 1 month for example.

Reply:
Ok, thanks for the clarification.